### PR TITLE
Enhanced ux

### PR DESCRIPTION
**Pull request by chxwcodes**
- made it so you cant scroll when the about modal is displayed
- h1s --> h2s on sign up/login components
- centered error modal and added click anywhere to close functionality
- gallery card icon styling
- working confirmation delete screen (asks user if they really want to delete before deleting)
👆needs styling lol ;_;

### CODE REVIEW

File: src/components/AboutModal.js [hunk 1]
```diff
@@ -24,7 +33,7 @@ const AboutModal = props => {
           </p> 
           <div className="aboutHowTo">
             <h2 className="howToTitle">How To Use Backronyms</h2>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus assumenda debitis distinctio architecto fuga magnam inventore placeat, nostrum repudiandae non doloribus aut magni rerum pariatur officia laborum quaerat tenetur laboriosam.</p>              
+            <p>First, enter a word that you want an acronym for. Our generator will first give you a random list of words for the first initial. Then, our generator will give you words that it think best suits your previously selected word. This will repeat until your backronym is created! Neat! Make sure you save your awesome backronym to our backronym gallery. Logged in users can be able to like and delete backronyms, so be sure to create an account!</p>
```
Comment (pinggyu) [inline src/components/AboutModal.js:36]: yay no more lorum!

File: src/components/DeleteConfirmation.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+function DeleteConfirmation({ setDeleteWarning, handleDelete, deleteID }) {
+  return (
+    <div className="errorModal" onClick={() => { setDeleteWarning(false) }}>
+      <div className="errorMsgContainer">
+        <p className="errorMessage">Are you sure you want to proceed? Changes are irreversible and your amazing backronym will be lost forever! &#x1F625;</p>
```
Comment (pinggyu) [inline src/components/DeleteConfirmation.js:7]: UX ENHANCED 🤯

File: src/components/ErrorModal.js [hunk 3]
```diff
@@ -7,8 +7,10 @@ function ErrorModal({ errorMsg, setError }) {
     }, 3000);
   }
   return (
-    <div className="errorModal">
-      <p className="errorMessage">{errorMsg}</p>
+    <div className="errorModal" onClick={() => { setError('') }}>
```
Comment (pinggyu) [inline src/components/ErrorModal.js:10]: yay compatible changes!

File: src/partials/_gallery.scss [hunk 4]
```diff
@@ -73,11 +69,54 @@
   font-weight: bold;
 }
 
+<<<<<<< HEAD
```
Comment (pinggyu) [inline src/partials/_gallery.scss]: remove '<<<<<<< HEAD' before merge

File: src/partials/_gallery.scss [hunk 5]
```diff
@@ -73,11 +69,54 @@
...
+  right: 15px;
+
+  .deleteBtn {
+    margin-bottom: 10px;
+  }
+  
+  .likeBtn {
+    margin-bottom: 3px;
+  }
+=======
```
Comment (pinggyu) [inline src/partials/_gallery.scss]: remove this line too

File: src/partials/_gallery.scss [hunk 6]
```diff
@@ -73,11 +69,54 @@
...
 .viewGalleryBtn {
   border: 2px solid #323031;
   box-shadow: 5px 5px 0px white;
+>>>>>>> 57d75b502539b81fc95b08649f208b1171824407
```
Comment (pinggyu) [inline src/partials/_gallery.scss]: remove this? what are those numbers ._.

**Review by pinggyu — CHANGES_REQUESTED** [review]
I think there are some lines from the conflict resolution interface to remove before merge?

**Comment by pinggyu** [comment]
@chxwcodes can you check if the code npm runs without errors before merging? there seems to be some lines shown on GitHub that need to be removed (e.g. <<<<<<< HEAD), pls ignore if they don't actually show up in your code, it's just showing up on GitHub

**Review by Uncle-Shred — APPROVED** [review]

**Review by pinggyu — APPROVED** [review]

**Pull request merged**